### PR TITLE
collection of release scripts.

### PR DESCRIPTION
**Pull request by dhatch**
PR checklist:

- [ ] Added changelog entry.

### CODE REVIEW

File: scripts/release/02-bump-versions [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+if [[ "$#" -ne 1 ]]; then
+  echo "$0: VERSION"
+  exit 1
+fi
+
+SCRIPTDIR=`dirname "$0"`
+VERSION="$1"
+pushd $SCRIPTDIR
+
+python ./bump_versions.py --oso_version "${VERSION}" \
```
Comment (gj) [inline scripts/release/02-bump-versions:13]: maybe fine to expect folks to have `packaging` & `tomlkit` installed (idk what's normal in Pythonland) but I had to install both when running this. for ✧･ﾟ: *✧･ﾟ:* extreme hands-off-ed-ness *:･ﾟ✧*:･ﾟ✧ we could pip install them here

File: scripts/release/03-push-tags [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+git tag -a "${OSO_TAG}" -m "${OSO_TAG}"
+git tag -a "${DJANGO_TAG}" -m "${DJANGO_TAG}"
+git tag -a "${FLASK_TAG}" -m "${FLASK_TAG}"
+git tag -a "${SQLALCHEMY_TAG}" -m "${SQLALCHEMY_TAG}"
+
+git push origin ${OSO_TAG}
+git push origin ${DJANGO_TAG}
+git push origin ${FLASK_TAG}
+git push origin ${SQLALCHEMY_TAG}
```
Comment (gj) [inline scripts/release/03-push-tags:32]: don't we want to wait for `OSO_TAG` to be pushed & published before pushing the others?

Comment (gj) [reply scripts/release/03-push-tags:32]: also sometimes we don't want to publish all 4 of these at once

File: scripts/release/04-publish-release [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+if [[ "$#" -ne 1 ]]; then
+  echo "$0: VERSION"
+  exit 1
+fi
+
+VERSION=$1
+
+function check_status () {
+  out=$(mktemp /tmp/osorelease.XXXXX)
+  RES=$(gh api -X GET repos/{owner}/{repo}/actions/runs -f "q='branch:v${VERSION}'" | \
```
Comment (gj) [inline scripts/release/04-publish-release:13]: looks like this requires folks to have the `gh` CLI installed? (I still use `hub` b/c I haven't bothered to update yet)

File: scripts/release/04-publish-release [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    jq ".workflow_runs[] | {name, id, conclusion, head_branch} | select((.name | contains(\"Release\")) and (.head_branch | contains(\"${VERSION}\")))" | tee "$out" | jq '.conclusion == "success"')
```
Comment (gj) [inline scripts/release/04-publish-release:14]: also requires folks to have `jq` installed — we have a check in one of our Makefiles to ensure that it's installed

File: scripts/release/bump_versions.py [hunk 5]
```diff
@@ -0,0 +1,225 @@
+"""Bump versions in files for release.
```
Comment (gj) [inline scripts/release/bump_versions.py:1]: How is this different from https://github.com/osohq/oso/blob/f89ae9d95678b68697c6dbfc736ac14ee89b09b8/scripts/bump_versions.py?

**Review by gj — CHANGES_REQUESTED** [review]
this looks freakin' great

File: scripts/release/03-push-tags (hunk 2 above)
Comment (dhatch) [reply scripts/release/03-push-tags:32]: even tho that's in the instructions. the other builds don't use the oso dep

File: scripts/release/bump_versions.py (hunk 5 above)
Comment (dhatch) [reply scripts/release/bump_versions.py:1]: same

File: scripts/release/04-publish-release (hunk 4 above)
Comment (dhatch) [reply scripts/release/04-publish-release:14]: yeah

File: scripts/release/04-publish-release (hunk 3 above)
Comment (dhatch) [reply scripts/release/04-publish-release:13]: yeah

File: scripts/release/02-bump-versions (hunk 1 above)
Comment (dhatch) [reply scripts/release/02-bump-versions:13]: yeah we could, it's required

Comment (dhatch) [reply scripts/release/02-bump-versions:13]: I think installing deps once is better than doing it every time

Comment (dhatch) [reply scripts/release/02-bump-versions:13]: also this is mega hack

File: scripts/release/bump_versions.py (hunk 5 above)
Comment (gj) [reply scripts/release/bump_versions.py:1]: rm one of the two & 🚢 ?

**Review by gj — APPROVED** [review]

**Comment by dhatch-niv** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by dhatch** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**